pkg/occlient/occlient.go

```diff
@@ -395,6 +406,31 @@ func (c *Client) GetPortsFromBuilderImage(componentType string) ([]string, error
 	return portList, nil
 }
 
+// RunBuildConfigWithBinary
+func (c *Client) RunBuildConfigWithBinaryInput(name string, r io.Reader) (*buildv1.Build, error) {
+	// TODO: investigate this issue
```

Comment:
Can you please add the issue link if any

Reply:
Unfortunately didn't find any issue online related to this problem